department-of-veterans-affairs/va.gov-cms
# VACMS-2393: Add all types to workflow.

## Description
See #2393 

## Testing done
Visual

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/89895047-27170f00-dba9-11ea-9c3e-8a2ef5c668c8.png)

## QA steps
As an admin, go to `/admin/config/workflow/workflows/manage/editorial` and verify that all content types are checked.
